src/components/stackview.js

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { html, LitElement } from "../vendor/lit-all.min.js";
+
+/**
+ * A Simple StackView
+ *
+ * Usage:
+ **<stack-view>
+ *  <other components/>
+ *  <lower/>
+ *  <currentComponent/>
+ * </stack-view>
+ *
+ *  Items can be added removed anytime using push(HtmlElement) or pop();
+ *
+ *  Note:
+ *  It will style the child-elements directly manipulating their .style attribures
+ *
+ *  TIP:
+ *  It does determine it's size from it's childs so making sure they are equally sized
```

Comment:
Checking my understanding: The StackView inherits its size from its child views so they should be equally sized? Also very neat.

Reply:
Yes. Kind of both ways. 
I chose display:grid with a 1x1 grid to lay them on top of each other. So size flows bottom up unless we define the size for the stackview.